openam-ui/openam-ui-ria/src/main/js/org/forgerock/openam/ui/user/login/RESTLoginView.js

```diff
@@ -417,6 +415,10 @@ define([
         handleUrlParams () {
             var urlParams = URIUtils.parseQueryString(URIUtils.getCurrentCompositeQueryString());
 
+            if (urlParams.goto) {
+                urlParams.goto = encodeURIComponent(urlParams.goto);
```

Comment:
This fix seems to break the process for SSORedirect.

see https://github.com/openam-jp/openam/commit/13422162b919a611421e551ab7e050e0a0a11376